python/pyspark/testing/utils.py

```diff
@@ -50,6 +51,44 @@ def write_int(i):
     return struct.pack("!i", i)
 
 
+def eventually(condition, timeout=30.0, catch_assertions=False):
```

Comment:
Just moved from `test_streaming_algorithms.py`

Reply:
minor -- maybe should delete the line "This is a helper utility for streaming ML tests"?